attestation.js

```diff
@@ -6,18 +6,46 @@ const db = require('byteballcore/db');
 const eventBus = require('byteballcore/event_bus');
 const validationUtils = require('byteballcore/validation_utils');
 const mail = require('byteballcore/mail');
+const emailjs = require('emailjs');
 const texts = require('./modules/texts');
 const reward = require('./modules/reward');
 const conversion = require('./modules/conversion');
 const emailAttestation = require('./modules/email_attestation');
 const notifications = require('./modules/notifications');
 const randomCryptoString = require('./modules/random-crypto-string');
+const i18nModule = require("i18n");
+const arrWhitelistEmails = Object.keys(conf.objRewardWhiteListEmails);
+let server;
+
+if (conf.bUseSmtp) {
```

Comment:
maybe you didn't know about these confs but seems like it duplicates what byteballcore/mail.js already does with conf.smtpTransport = 'relay'

Reply:
I am not sure what you mean, but I got an sendmail error even when i set conf.bUseSmtp to true.

Reply:
you probably got the error because the default transport is /usr/sbin/sendmail which most likely was not installed. Use the core which already handles different transports rather than notification.  Here is what the bot's conf.json looks like:
```
        "smtpTransport": "relay",
        "smtpRelay": "_server_address_",
        "smtpUser": "_user_",
        "smtpPassword": "_pass_",
```

Reply:
But that still doesn't seem to enable me to configure SMTP port (i think my ISP was blocking the default port) and ssl (default is on, but i have something not configure so I needed to disable on my local machine).

Reply:
the default is port 587 with STARTTLS, is this port blocked? port 465 is deprecated but we can add options to core conf if necessary

Reply:
Ok, then maybe it's was not probably even ISP because Mailtrap only has 25 or 465 or 2525, but I can commit a change that makes port 587 as default if conf.smtpPort is not set.